### PR TITLE
add roles to workspace in vs code as use 2 space for tab #15

### PR DESCRIPTION
**Pull request by ahmadysalah**
Vscode setting for the workspace.
its basic rolls on vscode like use one quota ensted to use double by defulte and two space for tab key ensted to use four space.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  // "editor.formatOnSave": true,
```
Comment (mFlaifel) [inline .vscode/settings.json]: what is the purpose of these two slashes?

Comment (Asmaathabet) [inline .vscode/settings.json]: remove double quotes from here

Comment (ahmadysalah) [reply .vscode/settings.json]: Its just we can use it if we want

Comment (Asmaathabet) [reply .vscode/settings.json]: ok, you have to change it for now to merge it

**Pull request merged**